net-vpn/i2pd/i2pd-2.15.0.ebuild

```diff
@@ -39,7 +39,6 @@ src_configure() {
 	mycmakeargs=(
 		-DWITH_AESNI=$(usex cpu_flags_x86_aes ON OFF)
 		-DWITH_HARDENING=$(usex i2p-hardening ON OFF)
-		-DWITH_PCH=$(usex pch ON OFF)
```

Comment:
If we are dropping PCH maybe you want to leave this on OFF.

Even if I2Pd fixes PCH support in the future and enables it is still known to cause issues on some systems (hence why the USE flag exists) and should be disabled by default.